src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
Can we avoid hard coding path like this because when we have multiple instances of WCF service hosted on one machine they will be overwritten by each other? Not sure if we can use relative path and save trace log in bin folder or something alike.